### ISSUE TITLE
[a11y] The context menu triggerred after text selection isn't read

### ISSUE THREAD

**Issue by lobontiumira** [mozilla-mobile/focus-android#6137]
## Steps to reproduce
1. Enable the talkback or the screen reader on your device.
2. Load a page in Focus, and trigger a context menu using TB gestures (long-tap with two fingers on a text).

### Expected behavior
The context menu triggerred after selecting a text is announced.

### Actual behavior
The context menu triggerred after selecting a text is not announced.
Tapping on one of the options (copy, select all, search privately), reads "pop-up window [title of the option]"

### Device information
* Android device: Google Pixel (Android 10)
* Focus version: Beta 96.0.0-beta.5, Focus Nightly 12/27
* Reproducible on Fenix Nightly 97.0a1 from 12/27

**Comment by ionutbedregeanu** [comment]
I didn't succeed to reproduce it on Focus Nightly 101, device Samsung A50 Android 11,  Talkback version 13.1.00.4

**Comment by lobontiumira** [comment]
I wasn't able to reproduce it either on the latest Focus Nightly 101.0a1 from 4/29, nor on Beta 100.0.0-beta.6.
Tested with Google Pixel (Android 10), and Oppo Find X3 Lite (Android 11), on the Talkback version 12.2.
I'll close this issue.
